regression/goto-harness/havoc-global-int-02/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
We can work around this for now by re-enabling the old behaviour for these, but we've marked them KNOWNBUG for now because we're first trying to get the call-function harness to behave like we want it to.